apps/test-suite/tests/FileSystem.ts

```diff
@@ -135,8 +135,14 @@ export async function test({ describe, expect, it, ...t }) {
 
     it('Allows reading files from assets', () => {
       const dir = new Directory(Paths.bundle);
-      expect(dir.list().map((i) => i.name)).toContain('Info.plist');
-      expect(new File(Paths.bundle, 'Info.plist').size > 2000).toBe(true);
+
+      if (Platform.OS === 'ios') {
+        expect(dir.list().map((i) => i.name)).toContain('Info.plist');
+        expect(new File(Paths.bundle, 'Info.plist').size > 2000).toBe(true);
+      } else {
+        expect(dir.list().map((i) => i.name)).toContain('expo-root.pem');
+        expect(new File(Paths.bundle, 'expo-root.pem').size > 1000).toBe(true);
+      }
```

Comment:
Could we create a file instead of relying on a file that is already there? It might change in the future and break our tests.

Reply:
tbh my thinking was that I want these tests to also pass in expo-go (for QA), and I don't want to add native asset to expo-go just for this single test, so I went with a file that is there on both bare-expo and expo-go